### PR TITLE
Elimination of level confusion

### LINKED ISSUE #300: confusion about tunneling or encapsulation of lower layeres
If encapsulation of UDP,IP is not in the scope of this specification then how do I interpret the following statements ?

https://github.com/tsvwg/ietf-multipath-dccp/blob/b8cc70de1761ddf446f1782fd8b0c728ac032a5d/draft-ietf-tsvwg-multipath-dccp.mkd?plain=1#L219-L220

### PR DESCRIPTION
**Pull request by markusa**
Closes #300

### CODE REVIEW

**Review by gorryfair — APPROVED** [review]
Looks OK

**Review by zaheduzzaman — COMMENTED** [review]
s/The protocol/MP-DCCP

**Comment by markusa** [comment]
> s/The protocol/MP-DCCP

Done, see https://github.com/tsvwg/ietf-multipath-dccp/pull/309/commits/5c4033770e20d05c22401036254c44ec1fe8f763

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 1]
```diff
@@ -409,7 +407,7 @@ shown in {{ref-option-list}}.
...
 
-The Multipath Capable feature (MP_CAPABLE) has feature number 10 and follows the structure for features given in {{RFC4340}} Section 6. Beside the negotiation of the feature itself, also one or several values can be exchanged. The value field specified here for the Multipath Capable feature has a length of one-byte and can be repeated several times within the DCCP option for feature negotiation if required for example to announce support of different versions of the protocol. For that, the leftmost four bits in {{ref-mp-capable-format}} specify the compatible version of the
+The Multipath Capable feature (MP_CAPABLE) has feature number 10 and follows the structure for features given in {{RFC4340}} Section 6. Beside the negotiation of the feature itself, also one or several values can be exchanged. The value field specified here for the Multipath Capable feature has a length of one-byte and can be repeated several times within the DCCP option for feature negotiation if required for example to announce support of different versions of ol. For that, the leftmost four bits in {{ref-mp-capable-format}} specify the compatible version of the
```
Comment (zaheduzzaman) [inline draft-ietf-tsvwg-multipath-dccp.mkd:410]: What is ths ol?

Comment (markusa) [reply draft-ietf-tsvwg-multipath-dccp.mkd]: Incorrect substitution reverted in https://github.com/tsvwg/ietf-multipath-dccp/pull/309/commits/327a8b7e554d35cbec5fe1a1ce34e0159362d883 and https://github.com/tsvwg/ietf-multipath-dccp/pull/309/commits/4cc65e8ccb3f2b8aee9a86006e4ceff7e47691ed.

**Pull request merged**